WordPress/wordpress-playground
# Add multisite rewrite rules

## What is this PR doing?

This PR adds support for multisite URL rewrites.

## What problem is it solving?

It ensures that all types of WordPress URLs work on Playground.

## How is the problem addressed?

By adding support for rewrite rules in PHP WASM and adding a rule to resolve WordPress multisite URLs.

## Testing Instructions

- Checkout this branch
- [Start a new multisite](https://playground.test/website-server/#{%20%22landingPage%22:%20%22/test/%22,%20%22phpExtensionBundles%22:%20[%22kitchen-sink%22],%20%22features%22:%20{%20%22networking%22:%20true%20},%20%22steps%22:%20[%20{%20%22step%22:%20%22enableMultisite%22%20},%20{%20%22step%22:%20%22login%22%20},%20{%20%22step%22:%20%22runPHP%22,%20%22code%22:%20%22%3C?php%20require_once%20'wordpress/wp-load.php';%20global%20$playground_scope;%20wp_insert_site(array('path'=%3E%20'/scope:'.$playground_scope.'/test/',%20'domain'=%3E%20parse_url(%20get_site_url(),%20PHP_URL_HOST%20),%20'user_id'=%3E1));%20?%3E%22%20}%20]%20}) (ensure playground.test proxy is running)
- Confirm that the subsite loaded
- Open WP-admin of that site and confirm that all assets loaded correctly
